back-end-side/Controllers/Sorting.cs

```diff
@@ -81,6 +81,25 @@ public static Boolean CheckType(Record record, String[] shopNames)
             }
             return false;
         }
+        public static String CheckInput(InputModel model)
```

Comment:
Have you thought about using LINQ?

Reply:
LINQ is already used by using .Any method for array :)